docker-compose.yml

```diff
@@ -50,7 +50,7 @@ services:
       - "9984"
     command: bigchaindb start
   
-  bdb-mdb:
+  bdb:
```

Comment:
The corresponding doc still has:
`docker-compose up -d bdb-mdb`
Link: https://docs.bigchaindb.com/projects/server/en/master/dev-and-test/setup-run-node.html